htdocs/js/jquery.quickreply.js

```diff
@@ -156,23 +159,19 @@ jQuery(function($) {
         }
     }
 
-    $("#submitpview").live("click", function(e){
+    $("#submitpview").on("click", function(e){
         $("#qrform input[name='submitpreview']").val(1);
         submitform(e);
     });
-    $("#submitpost").live("click", function(e){
-        var maxlength = 16000;
-        var length = $("#body").val().length;
-        if ( length > maxlength ) {
-            alert("Sorry, but your comment of " + length + " characters exceeds the maximum character length of " + maxlength + ". Please try shortening it and then post again");
-
+    $("#submitpost").on("click", function(e){
+        if ( checkCommentLength() ) {
```

Comment:
In the cold light of day I’m disliking this. I don’t think the helper code should export a global function. It should probably be a jquery extension, or better yet just attach it to the form submission event during the helper script. 

Is there a reason why the submit event might be too late for that, and we’d need to do it on click instead?

Reply:
Yes: on last-n pages, the quickreply submits as an ajax POST request, not a real form submission, and it does it in a goofy function called from the submit buttons instead of a submit handler. well, that's fixable.